Core/src/org/sleuthkit/autopsy/logicalimager/dsp/LogicalImagerPanel.java

```diff
@@ -75,10 +79,35 @@
      */
     private LogicalImagerPanel(String context) {
         initComponents();
+        configureImageTable();
         jScrollPane1.setBorder(null);
         clearImageTable();
     }
 
+    /**
+     * Perform the Image Table configuration necessary when a new table model is
+     * set.
+     */
+    private void configureImageTable() {
+        //hide path column while leaving it in model
+        if (imageTable.getColumnCount() > NUMBER_OF_VISIBLE_COLUMNS) {
+            TableColumn columnToHide = imageTable.getColumn(imageTableModel.getColumnName(NUMBER_OF_VISIBLE_COLUMNS));
+            if (columnToHide != null) {
+                imageTable.removeColumn(columnToHide);
+            }
+            //Allow refreshing of the Path displayed when sort order changed
+            imageTable.getRowSorter().addRowSorterListener(new RowSorterListener() {
+                @Override
+                public void sorterChanged(RowSorterEvent e) {
+                    imageTableSelect();
+                }
+            });
+            //sort on specified column in decending orders
+            imageTable.getRowSorter().toggleSortOrder(COLUMN_TO_SORT_ON_INDEX);
+            imageTable.getRowSorter().toggleSortOrder(COLUMN_TO_SORT_ON_INDEX);
```

Comment:
Duplicated code?

Reply:
the first one toggles it to ascending order, the second to descending I will make the comment more clear in that regard.